airgun/entities/subscription.py

```diff
@@ -22,7 +22,7 @@ def _wait_for_process_to_finish(self, name, has_manifest=False):
         :param name: Name of running task
         :param has_manifest: Should manifest exist after task ended?
         """
-        view = self.navigate_to(self, 'All')
+        view = SubscriptionListView(self.browser, logger=self.browser.logger)
```

Comment:
Navigate triggers `pre_navigate()`, which ensures fade widget is gone before proceeding. The thing is that fade is appearing when task is started and is usually gone when task is finished (pretty often *after* progress bar disappeared and flash notifications shown), so we can't wait till fade is gone as the next verifications may fail.

Reply:
hmm, so why do we need complex wait mechanism if we apply that 'not so elegant' workaround anyway?